media-sound/alsamixer-app/alsamixer-app-0.1.ebuild

```diff
@@ -1,16 +1,16 @@
 # Copyright 1999-2018 Gentoo Foundation
```

Comment:
Looks good, though this should go in a revision bump. Especially with 7 EAPI bumps, thanks for dusting the cobwebs off!

Reply:
No Problem :) I've renamed the ebuild (r1) and rebased the PR.

Reply:
In that case yes it is enough (tested the new revbump, no changes that should break stuff).

Usually though even for ~arch I keep the previous version around for a short time, in case it breaks something I had not seen. But I somehow think it should be fine here!